Feature: audiobook/podcast skip controls

andykelk opened the pull request:
Adds configurable skip controls for audiobooks and podcasts with user-selectable skip amounts of 10, 15, 30, or 60 seconds.

 **Features**
-  Forward and backward skip buttons appear only when playing audiobooks or podcasts
-  Users can choose skip duration (10/15/30/60 seconds) in Settings > Audiobooks & Podcasts
-  Rapid clicking accumulates skip time instead of jumping to the same position
-  Works across mini player, fullscreen player, and browser media session controls
-  Automatically detects audiobooks, podcasts, and podcast episodes

 **Implementation Details**

-  Skip buttons positioned between previous/next and play controls
-  Generic fast-forward/rewind icons
-  Shared SkipControlManager handles throttling logic similar to existing media session controls
-  Browser media session uses configured amounts for keyboard/headphone controls
-  Falls back to 10 seconds for music content to maintain existing behaviour

 **Test Plan**
-  Skip buttons appear only for audiobooks and podcasts
-  Setting configurable in frontend settings under new "Audiobooks & Podcasts" category
-  Skip amount changes are applied immediately without page refresh
-  Rapid clicking accumulates skip time correctly
-  Browser media session (keyboard/headphone controls) uses configured amounts
-  No skip buttons shown for regular music content

 Fixes https://github.com/orgs/music-assistant/discussions/3745

marcelveldt commented:
Actually, server-side we also support skipping (for all media items, so also for tracks) - we simply never added it to the frontend. So if you prefer to use that, this would be the syntax:

```
command: "player_queues/skip"
parameters: {
  queue_id: string,
  seconds: number
}

seconds: number of seconds to skip in track. Use negative value to skip back.
```

tsipinakis commented:
FYI I've patched the build errors: https://github.com/tsipinakis/mass-frontend/tree/nikos if you want to pull them

OzGav commented:
@andykelk If you can fix the lint errors this can be progressed

andykelk committed:
feat: add configurable skip controls for audiobooks and podcasts

Adds skip forward/backward buttons to the player controls and fullscreen
view when the current media is an audiobook or podcast. The skip amount
is configurable via frontend settings (default 30s).

The buttons use the server-side player_queues/skip command, which also
works for tracks. The browser media session seek handlers use the same
configurable amount.
